File: Python/RS01DNA.py
def count_dna_nucleotides(string):
    A = string.count("A")
    C = string.count("C")
    G = string.count("G")
    T = string.count("T")
    return A, C, G, T

def main():
    string = "AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC"
    CNT_A, CNT_C, CNT_G, CNT_T = count_dna_nucleotides(string)
    print(CNT_A, CNT_C, CNT_G, CNT_T)

if __name__ == "__main__":
    main()

File: Java/RS01DNA.java
public class RS01DNA {
    public static void main(String[] args) {
        String DNA = "AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC"; 
        int CNT_A = 0, CNT_C = 0, CNT_G = 0, CNT_T = 0;
        char temp;
        for (int i = 0; i < DNA.length(); i++) {
            temp = DNA.charAt(i);
            if (temp == 'A') {
                CNT_A++;
            } else if(temp == 'C') {
                CNT_C++;
            } else if(temp == 'G') {
                CNT_G++;
            } else if(temp == 'T') {
                CNT_T++;
            }
        }

        System.out.print("A: " + CNT_A + "\nC: " + CNT_C + "\nG: " + CNT_G + "\nT: " + CNT_T + "\n");
    }
}